content/tutorials/reactor-firehose-kinesis.textile

```diff
@@ -0,0 +1,604 @@
+---
+title: Using Reactor Firehose with Amazon Kinesis
+alt_title: Using Reactor Firehose with Amazon Kinesis
+excerpt: Learn how to use Reactor Firehose to stream Ably messages into Amazon Kinesis and process them using an AWS Lambda and Ably pub/sub
+section: tutorials
+category:
+    - channels
+    - reactor-integrations
+group: sdk
+index: 120
+platform: browser
+authors:
+- author_name: Mark Lewin
+  author_bio: ""
+  author_profile_url: "https://github.com/marklewin"
+  author_image: "https://avatars.githubusercontent.com/u/18176755?s=460&v=4"
+level: medium
+reading_time: 20
+languages:
+    - nodejs
+ably_product: reactor
+tags:
+    - Reactor Integration
+    - AWS Lambda
+---
+
+"Reactor Firehose":/general/firehose enables "Enterprise customers":/pricing#enterprise to send messages published by the Ably platform directly to a "third party streaming or queuing service":/general/firehose#services. 
```

Comment:
fixed

Reply:
> "Enterprise customers":/pricing#enterprise 

Still 404. The reason is because `/foo/bar` will be relative to `/documentation` - this will need to be an absolute link as it's not relative to `/documentation`.